build/fpga-regmap/src/lib.rs

```diff
@@ -155,19 +163,22 @@ fn write_reg_fields(children: &[Node], prefix: &str, output: &mut String) {
 {prefix}        pub const {inst_name}: u8 = 0b{mask:08b};",
             )
             .unwrap();
+
             // Deal with optional encoded Enums on this field
             match encode {
                 Some(x) => {
+                    let encode_name = inst_name.clone() + "_Encoded";
```

Comment:
nitpicky, take it or leave it: the use of `+` on strings is pretty uncommon in ideomatic Rust. i probably would've written this, instead:
```suggestion
                    let encode_name = format!("{inst_name}_Encoded");
```
as a side note, this has the advantage of only allocating once, while `inst_name.clone() + "_Encoded"` will (likely) allocate twice, once for the clone and then again if the cloned string doesn't have enough capacity to add "_Encoded". this doesn't actually matter very much here, i'm sure that the impact of a second allocation on build times is basically un-noticeable, but i felt like i'd mention it anyway.

Reply:
Thanks! Feel free to nit away, there's always stuff to learn!